core/js/src/main/scala-3/org/log4s/package.scala

```diff
@@ -0,0 +1,5 @@
+package org.log4s
+
+final val Log4sConfig: log4sjs.Log4sConfig = log4sjs.Log4sConfig
+final val AllThreshold = log4sjs.LogThreshold.AllThreshold
+final val OffThreshold = log4sjs.LogThreshold.OffThreshold
```

Comment:
I think if you want to have it here, you need to move the following two files to `core/jvm/src/main/scala-3/org/log4s/package.scala`
* `core/shared/src/main/scala-3.0.0-M1/org/log4s/package.scala`
* `core/shared/src/main/scala-3.0.0-M2/org/log4s/package.scala`

So leave only one and move from `shared` to `jvm`.